tests/src/GC/Stress/testmix_gc.config

```diff
@@ -37,7 +37,7 @@
             arguments="" 
             concurrentCopies="1" 
     />
-   <Assembly id="Concurrentspin2.exe" 
+   <Assembly id="concurrentspin2.exe" 
             successCode="100" 
             filename="Concurrentspin2.exe" 
```

Comment:
Missed the error here

Reply:
@sdmaclea Oh strange.. I thought I already fixed that. Thanks for catching that.